src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/signin/SignInPage.java

```diff
@@ -0,0 +1,23 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin;
+
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword.ForgotPasswordPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register.RegisterPage;
+
+public interface SignInPage extends FormPage {
+
+  String pageTitle = "Sign in";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)